src/firefly/js/api/ApiViewer.js

```diff
@@ -1,6 +1,13 @@
 /*
  * License information at https://github.com/Caltech-IPAC/firefly/blob/master/License.txt
  */
+
+
+/**
+ * @public
+ * @summary Build the interface to remotely communicate to the firefly viewer
+ * @namespace firefly
+ */
```

Comment:
We might want to omit the namespace declaration here, because it's already declared in ApiHighlevelBuild. Having 2 identical namespace declarations causes duplicate firefly section in jsdoc/firefly.html